specification/ai/HealthInsights/HealthInsights.TrialMatcher/model.trialmatcher.tsp

```diff
@@ -109,7 +109,7 @@ model TrialMatcherInference {
 }
 
 @doc("""
-The clinical trials that the patient(s) should be matched to. 
+The clinical trials that the patient(s) should be matched to.
```

Comment:
I recommend moving any doc changes to a separate PR.

Reply:
VSCode trimmed the extra space, I didn't want to reconfigure my VSCode just for a space. If it's really something we hate, I can do it though